Packs/Jira/Integrations/JiraV3/JiraV3.py

```diff
@@ -0,0 +1,4381 @@
+                        'body': comment_body
+                    }
+                    client.add_comment(issue_id_or_key=remote_id, json_data=payload)
+            demisto.debug('Updated the entries (attachments and/or comments) of the remote system successfully')
+    except Exception as e:
+        demisto.error(f"Error in Jira outgoing mirror for incident {remote_args.remote_incident_id} \n"
+                      f"Error message: {str(e)}")
+    finally:
+        return remote_id
+
+
+def map_v2_args_to_v3(args: Dict[str, Any]) -> Dict[str, Any]:
+    """As part of keeping Jira V3 backwards compatible, this function is in charge of mapping
+    the command arguments of Jira V2 to the command arguments of Jira V3, since the command arguments
+    of Jira V2 were inconsistent.
+    """
+    v2_args_to_v3: Dict[str, str] = {
+        'startAt': 'start_at',
+        'maxResults': 'max_results',
+        'extraFields': 'fields',
+        'getAttachments': 'get_attachments',
+        'expandLinks': 'expand_links',
+        'issueJson': 'issue_json',
+        'projectKey': 'project_key',
+        'issueTypeName': 'issue_type_name',
+        'issueTypeId': 'issue_type_id',
+        'projectName': 'project_name',
+        'dueDate': 'due_date',
+        'parentIssueKey': 'parent_issue_key',
+        'parentIssueId': 'parent_issue_id',
+        'attachmentName': 'attachment_name',
+        'globalId': 'global_id',
+        'applicationType': 'application_type',
+        'applicationName': 'application_name',
+
+    }
+    v3_args: Dict[str, Any] = {}
+    demisto.debug(f'Got the following command arguments: {args}')
+    for arg, value in args.items():
+        if arg in ['issueId', 'issueIdOrKey']:
+            # In v2, there was no differentiation between issue id and key arguments,
+            # and in v3, we do differentiate.
+            if is_issue_id(value):
+                v3_args['issue_id'] = value
+            else:
+                v3_args['issue_key'] = value
+        elif arg in v2_args_to_v3:
+            v3_args[v2_args_to_v3[arg]] = value
+        else:
+            v3_args[arg] = value
+    return v3_args
+
+
+def main() -> None:
+    params: Dict[str, Any] = demisto.params()
+    # args: Dict[str, Any] = demisto.args()
+    args = map_v2_args_to_v3(demisto.args())
+    verify_certificate: bool = not params.get('insecure', False)
+    proxy = params.get('proxy', False)
+    # Cloud configuration params
+    cloud_id = params.get('cloud_id', '')
+    client_id = params.get('client_id', '')
+    client_secret = params.get('client_secret', '')
+    callback_url = params.get('callback_url', '')
+
+    # OnPrem configuration params
+    server_url = params.get('server_url', 'https://api.atlassian.com/ex/jira')
+
+    # Fetch params
+    issue_field_to_fetch_from = params.get('issue_field_to_fetch_from', 'id')
+    fetch_query = params.get('fetch_query', 'status!=done')
+    id_offset = params.get('id_offset', '0')
```

Comment:
Let's add the initial value 0 to the yml as well. 
In addition it should be an int not a string.